src/lib/YouTube.ts

```diff
@@ -331,7 +336,7 @@ export class YouTube {
         (artist) =>
           artist.name.toLowerCase() === track.artists[0].name.toLowerCase()
       ) ||
-      compareTwoStrings(res.title ?? res.name ?? "", track.name) < 0.6
+      jaroWinkler(res.title ?? res.name ?? "", track.name) < 0.6
```

Comment:
You should test this line. Jaro Winkler matches differently so you may need to tweak the `0.6`.

Reply:
I'm comparing the two strings just to make sure that the song found isn't something completely different (which can easily happen, thanks YouTube iAPI)